.github/instructions/LOCALE-MAINTENANCE.instructions.md

```
@@ -0,0 +1,707 @@
+// ✗ WRONG - key name doesn't match English
+"tabSintonizacion": { "message": "..." }  // Key must be "tabPidTuning"
+
+// ✗ WRONG - translated key name
+"pestanaAjustePID": { "message": "..." }  // Key must stay "tabPidTuning"
+```
+
+---
+
+## Phase 2: Translation Rules
+
+## Phase 2: Translation Rules
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)